src/pages/verify/e-ids/swedish-bankid.mdx

```diff
@@ -60,6 +60,142 @@ For more information on how to configure your device for test and setup test use
+
+HTTP POST https://YOUR_DOMAIN.criipto.id/ciba/bc-authorize
+Content-Type: application/x-www-form-urlencoded
+
+scope=openid
+&callInitiator=RP
+&login_hint=sub:ssn:SSN
+&acr_values=urn:grn:authn:se:bankid
+&binding_message=BINDING_MESSAGE
+&client_assertion_type=urn:ietf:params:oauth:client-assertion-type:jwt-bearer
+&client_assertion=$client_assertion
+```
+
+2. A successful response will look something like this:
+
+```javascript
+{ 
+  "auth_req_id" : "3857f8ff-21b9-48ae-a732-a3bd8128a7ae",
+  "expires_in" : 120 
+}
+```
+
+3. Poll the token endpoint (`/oauth2/token`) to get a response with an ID token.  
+   The `auth_req_id` is used to construct a poll request on the token endpoint:
+
+```javascript
+# Replace `AUTH_REQ_ID` with the id returned by authentication response
+# Replace `$client_assertion` with your JWT assertion
+
+HTTP POST https://YOUR_DOMAIN.criipto.id/oauth2/token
+Content-Type: application/x-www-form-urlencoded
+
+auth_req_id=AUTH_REQ_ID
+&grant_type=urn:openid:params:grant-type:ciba
+&client_assertion_type=urn:ietf:params:oauth:client-assertion-type:jwt-bearer
+&client_assertion=$client_assertion
+```
+
+**Polling Frequency**
+
+Bear in mind, that the authentication process requires interacting multiple times with the end-user, so the identification process could take some time. Consequently, the token endpoint should be polled every 5 seconds.
+
+4. The result of a poll request will depend on the state of the process:
+
+* On successful user identification, a successful token response is returned, e.g.:
+
+```javascript
+eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiIsImtpZCI6IjgyN0Q5QTNFOTg2MTY0OTVBQzZGRTE3MUFFNkRBM0IzQ0ExNDE5MjEifQ.eyJpc3MiOiJodHRwczovL25hdGFsaWEtZGV2LXRlc3QuY3JpaXB0by5pbyIsImF1ZCI6InVybjpjaWJhIiwiaWRlbnRpdHlzY2hlbWUiOiJzZWJhbmtpZCIsImF1dGhlbnRpY2F0aW9udHlwZSI6InVybjpncm46YXV0aG46c2U6YmFua2lkIiwiYXV0aGVudGljYXRpb25tZXRob2QiOiJ1cm46b2FzaXM6bmFtZXM6dGM6U0FNTDoyLjA6YWM6Y2xhc3NlczpTb2Z0d2FyZVBLSSIsImF1dGhlbnRpY2F0aW9uaW5zdGFudCI6IjIwMjQtMDUtMjFUMDg6NDg6NTcuMDM2WiIsIm5hbWVpZGVudGlmaWVyIjoiYmI5YzIzNjRkZmFlNDRmM2JjZmQ5OTkwNTNkOTRmOWUiLCJzdWIiOiJ7YmI5YzIzNjQtZGZhZS00NGYzLWJjZmQtOTk5MDUzZDk0ZjllfSIsInNlc3Npb25pbmRleCI6IjQ1OWJiMzEzLTlkYTAtNDc3MS04ZTBiLTczMzEwMjA2ZDkyOSIsInNzbiI6IjE5NjgwMjAyMDU3NSIsIm5hbWUiOiJUZXJuZSBQYXVsc2VuIiwiZ2l2ZW5uYW1lIjoiVGVybmUiLCJnaXZlbl9uYW1lIjoiVGVybmUiLCJzdXJuYW1lIjoiUGF1bHNlbiIsImZhbWlseV9uYW1lIjoiUGF1bHNlbiIsImlwYWRkcmVzcyI6IjE4NS4xNTcuMTM0LjEzNCIsImNvdW50cnkiOiJTRSIsImlhdCI6MTcxNjI4MTMzNywibmJmIjoxNzE2MjgxMzM3LCJleHAiOjE3MTYyODI1Mzd9.DtJGqyFSGY9jCHCO_o4YB6XdkOXQRJO_lLDPF48zPakeepwhONRAEI4R3EuXZRgK9wEg6PPi_LCuK8tT9A7vibMKkW7s15ZxylAu0kL95dzaVU6rVjg5327tBRl2pK1yrGQNwZYiyTPy8897LyaGBV0UwzTWM4Quf5Tezh-jUawoIg9s6oe4GlZfS3fkEra9cbu2sMogYXejRnRh033I6MRYXSnme8fU5dP0s8qZJj_EW0M3D-70TL67BZLOrQEfLbVdJD3MKttfOgkaWpBoR4qJSlITmINmAfH_Ouj4xp-y956NkYkYnmQaHu3dZqMwqjPs3R1IHT4tbo8XgwmJXQ
+```
```

Comment:
Is it returned direcetly in the payload? Is it not wrapped in an object? @kasperhj

Reply:
Returns an object – updated, thanks @mickhansen